src/file/maps.h

```diff
@@ -131,23 +144,25 @@ static __always_inline incomplete_file_message_t* set_path_mnt(struct pt_regs *c
 
  EmitWarning:;
     file_message_t fm = {0};
-    push_file_warning(ctx, &fm, event->kind);
+    push_file_warning(ctx, &fm, event->kind, probe_id);
     return NULL;
 }
 
-static __always_inline void set_current_file_mnt(struct pt_regs *ctx, void *file)
+static __always_inline void set_current_file_mnt(struct pt_regs *ctx, void *file, u64 probe_id)
 {
     u64 pid_tgid = bpf_get_current_pid_tgid();
     incomplete_file_message_t *event = bpf_map_lookup_elem(&incomplete_file_messages, &pid_tgid);
     if (event == NULL || event->vfsmount != NULL) return;
 
     void *path = read_field_ptr(file, CRC_FILE_F_PATH);
     if (path == NULL) goto EmitWarning;
-    set_path_mnt(ctx, event, path);
+    set_path_mnt(ctx, event, path, probe_id);
+
+    return;
```

Comment:
this missing `return` here was the cause of one of the bugs were we were sending warnings without a warning code. In reality, there was no warning at all to be sent.